docs/config.md

```diff
@@ -0,0 +1,40 @@
+# Configuration Keys
+
+Subkeys are formatted as `key.subkey` .
+
+e.g.
+
+```yaml
+things:
+  hello:
+```
+
+Will show up here as `things.hello`
+
+| Key                                          | Type             | Description                                                                                                                                                                                                                                                                                         |
+| -------------------------------------------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `homeserverUrl`                              | string           | Endpoint URL that Mjolnir uses to interact with the matrix homeserver, set to the pantalaimon URL if you're using that.                                                                                                                                                                             |
```

Comment:
I agree that this is some duplication, but my intention is for this file to be a more friendly and less technical reference than `default.yaml` is. (As that is meant to be more like a "form" to fill out)

Opinions differ though; My intention is for this to become a reference file. If that's redundant, I can remove it.

Reply:
Context: We agreed we should probably move the extra information from the table to `default.yaml` to avoid duplication and also because `default.yaml` should be able to explain itself as well as the table.